src/buyingcatalogue/templates/test-result-local-storage.yml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.localTestResultStorage.enabled -}}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: bc-test-result-pv
+  labels:
+    type: local
+spec:
+  storageClassName: hostpath
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/data/buyingcatalogue/test"
+---
+apiVersion: v1

```

Comment:
I think I'd like to get the local/remote distinction out of the main template.
Really, we need a single persistent volume claim, and the only thing that's changing is it's using hostpath locally, and azurefile remotely.
I think a value entry for 'testStorageClassName' might make more sense, and set it to hostpath in local-docker.yaml, and azurefile in azure.yaml

Reply:
That is an excellent suggestion, will do.